### DISCUSSION TITLE
Reload configuration

### DISCUSSION THREAD

**Discussion by bmallya** [vectordotdev/vector#10439] [General]
Hello,

As mentioned in the `usage` of vector, it is only looking at the file passed with the `-w` command to check for updates in configuration to reload, but no mention of checking other files (say in the same directory).

In my configuration, I have a `vrl` file associated with a `remap` transform and wanted to update some of the fields at run time. In a test that I ran, the change  I made in the `vrl` file were not reloaded.
Would you guys think that this is possible?

Thanks

**Comment by jszwedko** [comment]
Hi @bmallya !

That is a current short coming of the watch functionality. We are tracking this here: https://github.com/vectordotdev/vector/issues/9819 . Let me know if that doesn't answer your question!

**Reply by jszwedko** [reply]
As a work around, I recommend using an external process like [`rerun`](https://github.com/alexch/rerun) to watch the files and restart Vector when they change.

**Reply by bmallya** [reply]
Thanks.
